### DISCUSSION TITLE
Eligiendo un encabezado o anuncio

### DISCUSSION THREAD

**Discussion by SaryNotfound** [Curcolor/COMMUNITY#6] [Q&A] [answered]
1. Descubre recursos de ayuda gratuita y oportunidades para apoyar.
2. Conecta con comunidades que ayudan y que necesitan tu ayuda.
3. Una comunidad para compartir recursos y apoyo mutuo.

**Comment by JUNIORRDSR** [comment]
De que pagina seria ese encabezado

**Reply by SaryNotfound** [reply]
En el inicio, cuando se abra el enlace por primera vez, debe salir ese encabezado o enunciado (aún no se como decirle)  para familiarizar a las personas sobre lo que se trata la página como tal, es como el "titulo" o "bienvenida" de la pagina

**Reply by JUNIORRDSR** [reply, marked as answer]
mmm creo que la primera estaría bien al reflejar de que se trata la pagina

**Comment by Curcolor** [comment]
Creo la primera es la mejor opción.
